examples/java/pom.xml

```diff
@@ -283,6 +287,34 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
We should make beam-runners-google-cloud-dataflow-java runtime scope and drop optional from beam-runners-direct-java